simulator/risc_v/riscv_instr.cpp

```diff
@@ -274,10 +333,18 @@ void RISCVInstr<T>::init( const RISCVTableEntry<MyDatapath>& entry)
     this->operation = entry.type;
     this->executor  = entry.function;
     this->opname  = entry.entry.name;
-    this->print_dst  = entry.dst == Dst::RD || entry.dst == Dst::CSR;
+    this->print_dst  = ( entry.dst != Dst::ZERO)
+                    && ( entry.dst != Dst::MEPC)
+                    && ( entry.dst != Dst::SEPC);
     this->print_dst2 = entry.dst == Dst::CSR;
-    this->print_src1 = entry.src1 == Src1::RS1;
-    this->print_src2 = entry.src2 == Src1::RS2;
+    this->print_src1 = ( entry.src1 != Src1::ZERO)
```

Comment:
Please wrap these conditions to a new function.

Reply:
Should I use "cstdarg" in order to create this function universal for entry.dst, entry.src1 and entry.src2?

Reply:
Or is it better to create one function for entry.src and another one for entry.dst?

Reply:
```c++
static bool print_dst( Reg dst)
{
    return src != Reg::SEPC && // ...
}

static bool print_src( Reg src)
{
    return src != Reg::SEPC && // ...
}
```